docs/extended_api/pipelines/pipeline_consumer_wait_prior.md

```diff
@@ -0,0 +1,64 @@
+---
+grand_parent: Extended API
+parent: Pipelines
+---
+
+# `cuda::pipeline_consumer_wait_prior`
+
+Defined in header `<cuda/pipeline>`:
+
+```cuda
+template <cuda::std::uint8_t Prior>
+__host__ __device__
+void cuda::pipeline_consumer_wait_prior(cuda::pipeline<thread_scope_thread>& pipe);
+```
+
+Let _Stage_ be the pipeline stage `Prior` stages before the current one
+  (counting the current one).
+Blocks the current thread until all operations committed to _pipeline stages_
+  up to _Stage_ complete.
+All stages up to _Stage_ (exclusive) are implicitly released.
+
+## Template parameters
+
+| `Prior` | The index of the pipeline stage _Stage_ (see above) counting up from the current one. The index of the current stage is `0`. |
+
+## Parameters
+
+| `pipe` | The thread-scoped `cuda::pipeline` object to wait on. |
+
+## Notes
```

Comment:
This needs to be changed to the `Expects` form to be consistent with the rest of the documentation.